pkg/steps/new_e2e_test.go

```diff
@@ -94,8 +94,8 @@ func TestGenPods(t *testing.T) {
 			}},
 		},
 	}}
-	step := newNewE2ETestStep(config, params, nil, &jobSpec, nil)
-	ret, err := step.genPods(steps)
+	step := newNewE2ETestStep(config, params, nil, "", &jobSpec, nil)
+	ret, _, err := step.genPods(steps)
```

Comment:
Shouldn't we be testing the output of this?

Reply:
Yes, tests are by no means comprehensive yet.